syne_tune/callbacks/hyperband_remove_checkpoints_score.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import numpy as np
+import logging
+
+from syne_tune.try_import import try_import_gpsearchers_message
+
+try:
+    from scipy.special import betainc
+except ImportError:
+    logging.info(try_import_gpsearchers_message())
+
+
+def _binomial_cdf(
```

Comment:
`BinomialEstimator` is not really about binomial distributions, it maintains Beta distributions, which are conjugate to binomial.

I think this method here is covered by the test I wrote, which would fail if the binomial CDF for small counts would be wrong. I'd also consider this being tested by `scipy`, as it is a simple function of that.

Reply:
OK makes sense.